pyproject.toml

```diff
@@ -39,34 +39,27 @@ test = [
     "pytest-cov",
     "pytest-jupyter[server]>=0.6.0"
 ]
+dev = [
+    "hatch"
+]
 
 [tool.hatch.version]
 source = "nodejs"
+path = "package.json"
 
 [tool.hatch.metadata.hooks.nodejs]
 fields = ["description", "authors", "urls"]
 
-[tool.hatch.build.targets.sdist]
-exclude = [".github", "binder"]
-
-[tool.hatch.build.hooks.version]
-path = "jupyter_chat/_version.py"
-
-[tool.hatch.build.hooks.jupyter-builder]
-dependencies = ["hatch-jupyter-builder>=0.5"]
-build-function = "hatch_jupyter_builder.npm_builder"
-
-[tool.hatch.build.hooks.jupyter-builder.build-kwargs]
-build_cmd = "build:prod"
-npm = ["jlpm"]
-
 [tool.jupyter-releaser.options]
-version_cmd = "hatch version"
+version_cmd = "../../scripts/bump-version.sh"
+python_packages = [
+    "packages/jupyter-chat-extension"
+]
 
 [tool.jupyter-releaser.hooks]
 before-build-npm = [
     "python -m pip install 'jupyterlab>=4.0.0,<5'",
-    "jlpm",
+    "YARN_ENABLE_IMMUTABLE_INSTALLS=0 jlpm",
```

Comment:
Looks like something might still not be behaving as expected.

Normally this env var should not be needed. Maybe we can track that in an issue to have a look later?

Reply:
Not sure, because there are 2 packages in the project, and one depends on the other (`@jupyter/chat-extension` depends on `@jupyter/chat`), but the bump occurs on both packages.